### PR TITLE
added minting receipts steps

### PR DESCRIPTION
**Pull request by Mik-TF**
# Related Issue

This PR is an answer to issue #376 

# Work Done

- Removed minting report FAQ
- Added minting receipts step to farmers book with new Alpha Minting tool version and pdf generation of minting receipts on the Farms section

### CODE REVIEW

File: src/farmers/3node_building/minting_receipts.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Minting Receipts Information
+
+The minting receipts contain the following information:
+
+- Minting: <minting_receipt_hash>
+- start: <start of minting period>
+- end: <end of minting period>
+- TFT: <TFT minted by the 3Node>
+- Clout Units: <3Nodes Resources>
```
Comment (xmonader) [inline src/farmers/3node_building/minting_receipts.md:42]: Cloud*

Comment (Mik-TF) [reply src/farmers/3node_building/minting_receipts.md]: Good catch. Updated.

File: src/farmers/3node_building/minting_receipts.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+The ThreeFold Alpha minting tool will present the following information for each minting receipt hash:
+
+- Node ID
+- Farm Name
+- Measured Uptime
+- Node Resources
+  - CU
+  - SU
+  - NU
+  - Cru
```
[the remark this answers is not in the document]
Comment (Mik-TF) [reply src/farmers/3node_building/minting_receipts.md]: Adjusted. Thanks for the review.

**Review by A-Harby — APPROVED** [review]

**Comment by A-Harby** [comment]
@Mik-TF Could you check conflicts?

**Pull request merged**